encfs/encfs.cpp

```diff
@@ -60,18 +62,21 @@ namespace rlog {
 class RLogChannel;
 }  // namespace rlog
 
-#ifndef MIN
-#define MIN(a, b) (((a) < (b)) ? (a) : (b))
-#endif
-
 #define ESUCCESS 0
 
 using namespace std;
 using namespace std::placeholders;
 using namespace rlog;
 using rel::Lock;
 
-#define GET_FN(ctx, finfo) ctx->getNode((void *)(uintptr_t) finfo->fh)
+#define GET_FN(ctx, finfo) ctx->getNode((void *)(uintptr_t)finfo->fh)
+
+// XAttr prefix replacement.
+// Replaces "com.apple." with "encfs_apl." when storing xattrs.
+// TODO: future versions of encfs should encrypt xattrs, making this
```

Comment:
skip this and go straight to xattr encryption?

Reply:
Yep.
Did we ever encounter (other) occurrences of the addressed issue ?

Reply:
Yes, in addition sounds like #70 is fixed since FUSE for OS X (2.8.2).
So I think we can close this PR yes.